cmd/alerts/silence/add_silence.go

```diff
@@ -75,7 +75,7 @@ func AddSilence(cmd *addSilenceCmd) {
 	}
 
 }
-func AddAllSilence(clusterID, duration, comment, username, clustername string, kubeconfig *rest.Config, clientset *kubernetes.Clientset) {
+func AddAllSilence(clusterID, duration, comment, username, clustername string, kubeconfig *rest.Config, clientset *kubernetes.Clientset) error {
```

Comment:
1. It looks like `clusterID` and `clustername` never get used by this function - can we blow them away?
2. Now that this function returns an error, we should make sure that any call sites are handling that error, such as line 70 in this file.